### ISSUE TITLE
Follicular lymphoma 

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6081]
**Preferred term label:**
Follicular lymphoma 

**Synonyms**
Follicle centre lymphoma

**Definition (free text, please give PubMed ID)**
An indolent B cell lymphoproliferative disorder of transformed follicular center B cells. Follicular lymphoma (FL) is characterized by diffuse lymphadenopathy, bone marrow involvement, splenomegaly and less commonly other extranodal sites of involvement.
PMID: 29314206 

**Parent term (use hpo.jax.org/app)**
Lymphoma

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 613024

**Comment by pnrobinson** [comment]
added as HP:0033125
